spec/index.html

```diff
@@ -197,7 +197,8 @@ <h3>Quoted Triples</h3>
         <code><a href="#grammar-production-subject">subject</a></code>,
         <code><a href="#grammar-production-predicate">predicate</a></code>, and
         <code><a href="#grammar-production-object">object</a></code>
-        preceded by <code>&lt;&lt;</code> and followed by <code>&gt;&gt;</code>.
+        preceded by <code>&lt;&lt;</code> (two less-than signs) and 
```

Comment:
Because this isn't a semantic use of "less-than", using the codepoint `U+003C` might be better.

Reply:
I'm fine with including the codepoints.

However, I would not replace the `{less|greater}-than sign` names with the codepoints, because these are the proper Unicode names for these characters, and both name and codepoint are included where similar notes are made elsewhere in these documents.

Reply:
```suggestion
        preceded by <code>&lt;&lt;</code> (two less-than signs, `U+003C`) and 
```

Reply:
Based on my suggestion in https://github.com/w3c/rdf-turtle/pull/34#issuecomment-1644517751.

```suggestion
        preceded by two concatenated <code>&lt;</code> characters each having the code point U+003C.
```